src/main/java/io/dropwizard/primer/auth/PrimerAuthorizationRegistry.java

```diff
@@ -45,11 +45,11 @@
 @Slf4j
 public class PrimerAuthorizationRegistry {
 
```

Comment:
Why do we need to rename these variables? Since these are already part of PrimerAuthorizationRegistry.java, no need to have prefix primer in variables

Reply:
reverted variable naming